### PR TITLE
Fixes #1

### LINKED ISSUE #1: a 1:1 implementation of cw3-fixed-multisig
Dev Notes:
The cw3 reference contract is not structured the way it's usually done on Secret. For Example, see:
https://github.com/CosmWasm/cw-plus/blob/main/contracts/cw3-fixed-multisig/src/lib.rs vs.
https://github.com/scrtlabs/secret-template/blob/master/src/lib.rs

set_contract_version and cw-storage-plus not available in secret

set_contract_version - we recommend just ignore this reference (remove it). you won't be able to migrate the contract but that's fine for now. also this will probably become available when secret updates to cw1.0
cw-storage-plus - this could be refactored out - technically

### PR DESCRIPTION
**Pull request by kent-3**

### CODE REVIEW

**Comment by kent-3** [comment]
I wasn't sure whether to call it SNIP3 or SW3 (SW = SecretWasm). It's not really a SNIP, and I thought SW might be better.

File: .github/actions/install-rust/main.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const child_process = require('child_process');
```
Comment (dannym-arx) [inline .github/actions/install-rust/main.js:1]: do you really need node for this? it can be done with just a shell script

File: src/contract.rs [hunk 2]
```diff
@@ -0,0 +1,1026 @@
...
+    _start_after: Option<String>,
+    _limit: Option<u32>,
+) -> StdResult<VoterListResponse> {
+    // Currently no use for start_after or limit 
+    // Returns the full list of voters
+    let voters = voters_list_read(&deps.storage).load()?;
+    Ok(VoterListResponse { voters })
+}
+
+// #[cfg(test)]
```
Comment (dannym-arx) [inline src/contract.rs]: I think that unit tests are very important for this, would you be able to make those tests work?

**Review by dannym-arx — COMMENTED** [review]
Great job, Kent, would you mind taking a look at the things I commented on?

File: src/contract.rs (hunk 2 above)
Comment (kent-3) [reply src/contract.rs]: I agree - I'll give that another go.

File: .github/actions/install-rust/main.js (hunk 1 above)
Comment (kent-3) [reply .github/actions/install-rust/main.js:1]: To be honest, I don't know anything about these test files. I just got them from the secret labs contract template.

**Comment by kent-3** [comment]
I've reworked the unit tests and all 50 of them pass :)

**Pull request merged**